docker/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+. /etc/.bashrc
+
+# # Run CMD
+# /bin/bash "$@"
+
+echo "$@"
+exec "$@"
```

Comment:
This entrypoint won't work as the goal is to have an interactive shell after setting up the environment with `. /etc/.bashrc`.